src/util/dump.ts

```diff
@@ -58,8 +58,7 @@ export async function getAndProcessStylesheets(downloader: Downloader, redisStor
               url = getFullUrl(url, cssUrl)
               url = url.indexOf('%') < 0 ? encodeURI(url) : url
 
-              /* Download CSS dependency, but avoid duplicate calls */
-              if (!downloader.cssDependenceUrls.hasOwnProperty(url) && filename) {
+              if (filename) {
                 downloader.cssDependenceUrls[url] = true
```

Comment:
I think this is wrong… there is no reason why we should allow to download twice the same url… but no time to make the full investigation. Not sure what this `cssDependenciesUrls` is exactly about… but either we should oeep it working like it was or this should be cleaned as well between to dumps.

Reply:
What about javascript dependencies?

Reply:
This PR is clearing Redis store after creating zim file. With one command the script can create more than one zim file only if you put more than one `--format` parameter. The `--format` parameter takes effect only on media files. So other files will be the same for each zim file. That's why we don't need to repopulate js files. They should be downloaded one time and used for each zim file.